frontend/packages/dev-console/src/components/topology/__tests__/topology-utils.spec.ts

```diff
@@ -221,3 +250,29 @@ describe('TopologyUtils ', () => {
     expect((topologyTransformedData[keys[0]].data as WorkloadData).editUrl).toBe(mockGitURL);
   });
 });
+
+describe('Topology Utils', () => {
+  beforeAll(() => {
+    jest
+      .spyOn(k8s, 'k8sCreate')
+      .mockImplementation((model, data) => Promise.resolve({ model, data }));
```

Comment:
will it resolve to  model and data both or just `data` as `createTopologyResourceConnection` should return ` Promise<K8sResourceKind[] | K8sResourceKind> ` and `K8sResourceKind` doesn't have `model`

Reply:
@invincibleJai yes, correct. It will resolve only the data. Fixed it.